pkgs/by-name/no/nomnatong/package.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchurl
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "nomnatong";
+  version = "5.07";
+
+  srcs = [
+    (fetchurl {
+      name = "nomnatong-regular.otf";
+      url = "https://github.com/nomfoundation/font/releases/download/v${finalAttrs.version}/nomnatong-regular.otf";
+      hash = "sha256-ojOh6sewZpcGqc8Kg+zeiu0g42MAnLYq1F1BWaFwc2s=";
+    })
+    (fetchurl {
+      url = "https://github.com/nomfoundation/font/releases/download/v${finalAttrs.version}/nomnatong-regular.ttf";
+      hash = "sha256-SCBYZ/A+NllhD+uMtr6WmM/vRbGkejiDxMCHhoaFhp8=";
+    })
+    (fetchurl {
+      url = "https://github.com/nomfoundation/font/releases/download/v${finalAttrs.version}/nomnatonglight.ttf";
+      hash = "sha256-A530sFooQjvMIscDU3DT7ZboUjGyN+TJY++mtKPUiUE=";
+    })
+  ];
+
+  sourceRoot = ".";
+
+  dontUnpack = true;
+
+  postInstall = ''
+    for i in ''${srcs[@]}; do
+      install -Dm444 $i $out/share/fonts/truetype/''${i#*-}
+    done
+  '';
+
+  meta = {
+    homepage = "http://nomfoundation.org/nom-tools/Nom-Font";
+    description = "The Hán-Nôm Coded Character Set and Nom Na Tong Regular Reference Font";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ eclairevoyant ];
+    platforms = lib.platforms.all;
+  };
```

Comment:
```suggestion
    sourceProvenance = [ sourceTypes.binaryBytecode ];
  };
```

Reply:
This isn't really "bytecode" is it? It's a font file

Reply:
But it's a prebuilt asset, and `binaryNativeCode` is an even worse fit.

Reply:
I've updated this to not use any prebuild assets.